src/style.css

```
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf") format("ttf");
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir-Heavy";
+  src: url("fonts/Avenir-Heavy.ttf") format("ttf");
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc") format("ttc");
+  font-weight: 800;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Avenir-Book", sans-serif, serif;
+  margin: 0;
+  padding: 20px 8px 0 7.5px;
+}
+
+.big_form {
+  margin-bottom: 20px;
+}
+
+.label {
+  display: inline-block;
+  width: 100%;
+}
+
+.input {
+  outline: none;
+  font-family: inherit;
+  display: inline-block;
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-weight: bold;
+  background: url("images/Search.svg") 25px 48% / 19px 19px no-repeat;
+  padding: 0 0 0 62.5px;
+  line-height: 68px;
+  font-size: 16px;
+}
+
+.small_input {
```

```suggestion
.small-input {
```

Due to [BEM](https://en.bem.info/methodology/quick-start/#modifier)